book/website/reproducible-research/open/open-notebooks.md

```diff
@@ -1,6 +1,14 @@
 (rr-open-notebooks)=
 # Open Notebooks
 
+```{figure} ../../figures/executable-notebooks.*
+---
+name: Executable Notebooks
+alt: basing a project around an open exceutable notebook such as a jupyter book is shown to allow praticipants to Understand the direction of the project, track its progress, Enable diverse participation, create transparentcy in outcomes and make auditing easier for organisers.
```

Comment:
This is a good start for alt text! I would also recommend adding in the following: 

- Description of style (cartoon) and colour scheme 
- Some more literal description of what is happening in the image (e.g. two people look at a easel chart labelled project, with the logos for GitHub and Jupyter Books underneath...) 

At the moment, people would be able to understand the purpose of the image but not how it's conveying this message and it's good to have both in the alt-text :)